### PR TITLE
feat: add 'Blockchain Developer Path' section

### PR DESCRIPTION
**Pull request by mkubdev**
## Related Issue 

N/A

Closes: # 

### Describe the changes you've made

I added a new section for the website : 'Blockchain Developer Path' and cards with an hover effect.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
Capture | new |

### CODE REVIEW

File: index.html
Comment (Panquesito7) [inline index.html:19]: Wouldn't this be the correct code?
```suggestion
    <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
```

File: index.html
Comment (Panquesito7) [inline index.html:87]: I think `Explore` sounds better. This doesn't have to be accepted, just a suggestion. 🙂
```suggestion
              Explore
```

File: index.html
Comment (Panquesito7) [inline index.html:158]:
```suggestion
          The latest field in the tech industry
```

File: index.html
Comment (mkubdev) [reply index.html]: I'm ok with your suggestion, think same. :)

File: index.html
Comment (mkubdev) [reply index.html:19]: fixed!

**Review by mkubdev — COMMENTED** [review]
fix added

**Review by Panquesito7 — APPROVED** [review]
All is looking awesome! It'd also be great if we can see the deployment of the live website. 🙂
Thank you for your work! I'll let @vinzvinci review this PR before merging, though.

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Pull request merged**
